list_port_forwards/display_port_forwards.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
bash has tricky error-handling. here's a good explanation:
https://wizardzines.com/comics/bash-errors/

I'd probably add
set -euo pipefail
here, which is closer to the semantics you want

Reply:
Thank you for the explanation!